docker/Dockerfile.dev

```diff
@@ -0,0 +1,16 @@
+FROM continuumio/miniconda3
+
+RUN apt install julia -y && apt-get clean
+COPY ./environment.yml /src/ESMValTool/environment.yml
+WORKDIR /src/ESMValTool
+RUN conda update -y conda pip && conda env update --name base --file environment.yml && conda clean --all -y
+
+COPY ./esmvaltool/install/R /src/ESMValTool/esmvaltool/install/R
+RUN Rscript ./esmvaltool/install/R/setup.R
+
+COPY . /src/ESMValTool
+RUN pip install -e . && pip cache purge && esmvaltool install Julia && pip uninstall esmvaltool -y
+RUN rm -rf /src
```

Comment:
```suggestion
```
or you will delete the just installed package?

Reply:
Here we can remove the package because this is the dev and we don't need ESMValTool code